windows-deps.h

```diff
@@ -4,33 +4,29 @@
 
 // winnt.h
 typedef void *HANDLE;
-typedef wchar_t         WCHAR;
-typedef const WCHAR    *PCWSTR,     *LPCWSTR;
+typedef wchar_t WCHAR;
+typedef const WCHAR *PCWSTR, *LPCWSTR;
 
-typedef uint8_t     BYTE;
-typedef uint32_t    DWORD;
+typedef uint8_t BYTE;
+typedef uint32_t DWORD;
 
-
-#define IMAGE_DLLCHARACTERISTICS_DYNAMIC_BASE           0x0040
-#define IMAGE_FILE_RELOCS_STRIPPED	                    0x0001
-#define IMAGE_DLLCHARACTERISTICS_FORCE_INTEGRITY        0x0080
-#define IMAGE_DLLCHARACTERISTICS_NX_COMPAT              0x0100
-#define IMAGE_DLLCHARACTERISTICS_NO_ISOLATION           0x0200
-#define IMAGE_DLLCHARACTERISTICS_NO_SEH                 0x0400
+#define IMAGE_DLLCHARACTERISTICS_DYNAMIC_BASE 0x0040
+#define IMAGE_FILE_RELOCS_STRIPPED 0x0001
+#define IMAGE_DLLCHARACTERISTICS_FORCE_INTEGRITY 0x0080
+#define IMAGE_DLLCHARACTERISTICS_NX_COMPAT 0x0100
+#define IMAGE_DLLCHARACTERISTICS_NO_ISOLATION 0x0200
+#define IMAGE_DLLCHARACTERISTICS_NO_SEH 0x0400
```

Comment:
Nit: Do we want to tell clang-format not to format these `#defines`, so that the hex numbers line up?

Reply:
I'm mostly basing this comment off of what appears in pe-parse here, but it's the comments that line up, not the hex 🤷‍♂  https://github.com/trailofbits/pe-parse/blob/master/pe-parser-library/include/parser-library/nt-headers.h#L38-L85

Reply:
I'd prefer it not force auto-format, since that can cause issues. Clang-format broke the build in #45 . We could just run clang-format manually.

Reply:
Hmm, no strong preference on my side. I'm planning on removing these `#define`s in a follow-up PR anyways, so they'll be short-lived 😄.